papermill/execute.py

```diff
@@ -233,15 +233,18 @@ def _translate_type_r(val):
     return _translate_escaped_str(val)
 
 _translate_type_str_scala = _translate_escaped_str
-_translate_type_int_scala = _translate_to_str
 _translate_type_float_scala = _translate_to_str
 
+def _translate_type_int_scala(val):
+    strval = _translate_to_str(val)
+    return strval + "L" if val > 2147483647 or val < -2147483648 else strval
```

Comment:
You could do `abs(val+.5) > 21474783647.5`, either that or I'd recommend putting parents around the condition for legibility.

Reply:
Should we have [BigInt](https://www.scala-lang.org/api/current/scala/math/BigInt.html) as a fallback for numbers longer than Long supports?

Reply:
Since these come from json spec inputs, https://tools.ietf.org/html/rfc7159#section-6 says Long will be sufficient.

Reply:
I'll wrap the conditional in parens. I find the two conditionals much easier to quickly grok than the `abs(val+.5) > 21474783647.5` where I'd be worried about floating rounding errors.

Reply:
I don't think there should be rounding errors when comparing two floats, but I agree that it's easier to quickly grok.